### DISCUSSION TITLE
Use labels to categorize the submitted issues

### DISCUSSION THREAD

**Discussion by TimJentzsch** [ValveSoftware/Dota2-Gameplay#932] [Ideas]
Since this is an issue related to the repository itself, I created this as a discussion (the issue templates don't fit here).

GitHub labels are a great way to further categorize the submitted issues.
Labels can be automatically added via the issue templates to make it easier to keep issues organized.

In fact, the current templates already define labels, but they need to be created first to be used:
- `ability-draft`: An issue related to the Ability Draft game mode.
- `ability`: An issue related to an ability.
- `cosmetic`: An issue related to cosmetic items, e.g. particle effects.
- `item`: An issue related to an item.
- `tooltip`: An issue related to tooltips.

For now I recommend to just create these labels with appropriate descriptions to make the issue templates work properly. At some point, select community contributors could also be given permissions to manage labels to help keep the issues organized.

**Possible use-cases**:
- An artist filters the issues by `cosmetic` to focus on issues relevant to their expertise.
- A software engineer filters the issues by `ability-draft`, `ability` and `item` to focus on issues relevant to their expertise.

**Comment by TimJentzsch** [comment]
Of course, other labels might also make sense to add.
For example an `intended` label could be used to mark issues as intended behavior before closing them, which could be useful to easily track those statistics.

**Comment by RGBKnights** [comment]
I have asking people to tag the title of an issue with [Ability Draft] this should be the process has a BOT automatically applying labels after the fact.

**Comment by onesmartsheep1** [comment]
Does tag and a category are the same things? I am new to the forum, so please help me. Thank u[.](https://onesmartsheep.com/)

**Reply by TimJentzsch** [reply]
Depends on what you mean by tag. The categories are only a thing for the discussion forums, they determine the rough topic that your entry is about and users can filter the entries by category. Labels are a general tool in GitHub to categorize things, they are also really useful, because you can filter issues by labels so they allow you to organize everything. If by tag you mean putting a text like `[Ability Draft]` in the title, then this is only a workaround to provide a bit more organization, but labels are definitely superior as you can use them for filtering.

Hope that helps :)
